var/spack/repos/builtin/packages/py-hepstats/package.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyHepstats(Package):
+    """hepstats is a library for statistical inference aiming
+    to cover the needs High Energy Physics.
+    It is part of the Scikit-HEP project.
+    """
+
+    homepage = "https://github.com/scikit-hep/hepstats"
+    pypi = "hepstats/hepstats-0.8.0.tar.gz"
+
+    maintainers("jonas-eschle")
+
+    license("BSD-3-Clause", checked_by="jonas-eschle")
+
+    tags = ["likelihood", "statistics", "inference", "fitting", "hep"]
+
+    version("0.8.1", sha256="78f283fc77b2a1bd0f6dd108f2ecce269359a14797cbf3a1af2eea9a29ce96db")
```

Comment:
This has the same issue and fix I suggested at https://github.com/spack/spack/pull/43696#discussion_r1572817991 plus the sha256 is incorrect.  

If you don't want to use `spack checksum` (https://spack.readthedocs.io/en/latest/command_index.html#spack-checksum) to confirm the sha256, pypi shows the value two ways:

- when you click the `view hashes` link to the right of the archive file (e.g., https://pypi.org/project/hepstats/0.8.1/#modal-close);
- it is appended to the end of the URL for the desired archive file at https://pypi.org/simple/hepstats/.

Reply:
Updated, thanks!

I'd love tho use `spack checksum` if it worked, it doesn't: instead, it's 
```
Error: Could not find any remote versions for py-hepstats
```
(yes, the pypi path was updated. Is this a typo in the path,  a feature or a bug of spack maybe?)

Reply:
I can now confirm the version sha256.